lib/test/monetary.test.ts

```diff
@@ -0,0 +1,162 @@
+import Big from 'big.js';
```

Comment:
This file should be moved to `test/lib/monetary.test.ts`.

Reply:
I would rather keep the library as self-contained as possible and have the library tests live with the library. What do you think?

Reply:
Moved as discussed during the call.